tests/infra/network.py

```diff
@@ -509,7 +513,7 @@ def wait_for_state(self, node, state, timeout=3):
     def _get_node_by_id(self, node_id):
         return next((node for node in self.nodes if node.node_id == node_id), None)
 
-    def find_primary(self, timeout=3, request_timeout=3):
+    def find_primary(self, timeout=3):
```

Comment:
We're still passing this arg in `election.py`, line 89.

Reply:
Good spot! This is now using `timeout`.